gazelle/src/com/google/idea/blaze/gazelle/GazelleSyncPlugin.java

```diff
@@ -0,0 +1,13 @@
+package com.google.idea.blaze.gazelle;
+
+import com.google.common.collect.ImmutableList;
+import com.google.idea.blaze.base.projectview.section.SectionParser;
+import com.google.idea.blaze.base.sync.BlazeSyncPlugin;
+
+import java.util.Collection;
+
+public class GazelleSyncPlugin implements BlazeSyncPlugin {
+    public Collection<SectionParser> getSections() {
+        return ImmutableList.of(GazelleSection.PARSER);
```

Comment:
- As far as I understand, the plugin settings are global, for all projects. For instance, the bazel binary you want to use tends to be the same, regardless of which project you're using. The Gazelle target, however, can vary from project to project. One repository might have it in `//:gazelle`, whereas another repo might store it under `//tools:gazelle`, or something different. Therefore, this configuration should be per-project. The plugin settings exist as a stopgap, so that one can start using the Gazelle functionality somewhat, before one's entire team has switched to the new version.

- Thanks! Will do.